pymc/distributions/timeseries.py

```diff
@@ -32,49 +37,94 @@
     "MvStudentTRandomWalk",
 ]
 
+class AR1rv(RandomVariable):
+    name = "AR1"
+    ndim_supp = 0
```

Comment:
```suggestion
    ndim_supp = 1
```

Reply:
i saw this comment on grw pr as well and for some reason ndim_supp = 1 throws an error when i add observed, something like "dimension mismatch of 2 and 1" as in data is 1d or something, i dont know why does it happen though

Reply:
Yes, it's related to the comments below, you haven't yet told Aesara how to infer the shape of the distribution given the parameters (and size). It tries to do something by default, but usually fails with multivariate distributions where there is not a simple 1-1 map between parameter shapes and distribution shape